src/mapbox/mapbox.js

```diff
@@ -221,12 +221,13 @@ export default class Mapbox {
     const map = this._map;
     // map render will throw error if style is not loaded
     if (map.isStyleLoaded()) {
-      map._render();
       // cancel the scheduled update
       if (map._frame) {
         map._frame.cancel();
         map._frame = null;
       }
+      // the order is important - render() may schedule another update
+      map._render();
```

Comment:
Correct me if my understanding is wrong. 

`map._render() will trigger another update` means in Mapbox's `render()` function will request another animation frame, which will be assigned to `map._frame`.

Reply:
Mapbox's `render` does not, but in the issue linked above, a custom layer does.